mapc_sim/sim.py

```diff
@@ -61,9 +61,11 @@ def network_data_rate(key: PRNGKey, tx: Array, pos: Array, mcs: Array, tx_power:
     sinr = sinr + tfd.Normal(loc=jnp.zeros_like(signal_power), scale=sigma).sample(seed=normal_key)
     sinr = (sinr * tx).sum(axis=0)
 
-    success_probability = tfd.Normal(loc=MEAN_SNRS[mcs], scale=2.).cdf(sinr) * (sinr > 0)
+    sdist = tfd.Normal(loc=MEAN_SNRS[mcs], scale=2.)
+    logit_success_prob = (sdist.log_cdf(sinr) - sdist.log_survival_function(sinr)) * (sinr > 0)
```

Comment:
Why do we need to subtract `log_survival_function`?

Reply:
Becouse $logit(p)=log(p/1-p)=log(p)-log(1-p)$

Reply:
True, you're right